pkg/csi_driver/node_test.go

```diff
@@ -557,6 +557,79 @@ func TestNodeGetInfo(t *testing.T) {
 func TestNodeGetCapabilities(t *testing.T) {
 }
 
+func TestNodeGetVolumeStats(t *testing.T) {
+	testEnv := initTestNodeServer(t)
+	ns := testEnv.ns
+
+	// Setup mount target path
+	tempDir, err := ioutil.TempDir("", "ngvs")
+	if err != nil {
+		t.Fatalf("failed to setup testdir: %v", err)
+	}
+	defer os.RemoveAll(tempDir)
+	targetPath := filepath.Join(tempDir, "mount")
+	stagingPath := filepath.Join(tempDir, "staging")
+
+	req := &csi.NodePublishVolumeRequest{
+		VolumeId:          testVolumeID,
+		TargetPath:        targetPath,
+		StagingTargetPath: stagingPath,
+		Readonly:          false,
+		VolumeCapability:  testVolumeCapability,
+		VolumeContext:     testVolumeAttributes,
+	}
+	_, err = ns.NodePublishVolume(context.Background(), req)
+	if err != nil {
+		t.Fatalf("Failed to set up test by publishing default vol: %v", err)
+	}
+
+	testCases := []struct {
+		name       string
+		volumeID   string
+		volumePath string
+		expectErr  bool
+	}{
+		{
+			name:       "normal",
```

Comment:
Can we also test out if the response matches with expectation.

Reply:
It will be flaky to test the response, since the file directory is created randomly, the stat of it will not be a fixed value for each test. 
IMO as long as this function didn't return an error, the response body must be valid.